src/main/java/org/folio/circulation/domain/CheckInLogRecord.java

```diff
@@ -0,0 +1,96 @@
+package org.folio.circulation.domain;
+
+import java.util.UUID;
+
+import org.apache.commons.lang3.ObjectUtils;
+import org.joda.time.DateTime;
+
+import io.vertx.core.json.JsonObject;
+
+public class CheckInLogRecord {
```

Comment:
Why is this named a log record rather than only a record? 

If we want to include the term log, I think we should also do so in the storage API.

Reply:
Fine for me to rename to `CheckInRecord`

Reply:
I think that is ok, given that we don't really have a convention for this kind of record.

(If the system was recording events, I'd likely name it `ItemCheckedInEvent`)